src/index.js

```diff
@@ -37,51 +36,47 @@ io.on('connection', async (socket) => {
     })
     if (error) throw error
 
-    const context = {
-        user,
-        room,
-    }
-
-    socket.join(context.room)
+    const context = { user, room }
+    socket.join(room)
     socket.emit('message', generateWelcome())
-    socket.to(context.room).emit('message', generateJoined(user))
+    socket.to(room).emit('message', generateJoined(user))
 
-    const roomWithUsers = await roomUsersFetcher.call(room)
-    io.to(room).emit('roomData', roomWithUsers)
+    const syncRoomData = async () => {
```

Comment:
Why is this method not cut out into own service/something like others?
Why is it defined here?

Reply:
It could be segregated. But it is depends on data transfer layer